### PR TITLE
respond to PING sent from client

### PR DESCRIPTION
**Pull request by Akuli**
On libera:

```
PING
:strontium.libera.chat 461 akuli-testing PING :Not enough parameters
PING :blah blah
:strontium.libera.chat PONG strontium.libera.chat :blah blah
```

Mentioned in #70

### CODE REVIEW

File: commands.py [hunk 1]
```diff
@@ -469,6 +469,14 @@ def handle_who(state: server.State, user: server.UserConnection, args: List[str]
     user.send_que.put((end_of_who_message, "mantatail"))
 
 
+def handle_ping(state: server.State, user: server.UserConnection, args: List[str]) -> None:
+    """Handles a PING message sent from a client to the server."""
```
Comment (ThePhilgrim) [inline commands.py:473]: Can we explain in the docstring why the PONG response looks like it does? (Have a look at the docstring of `handle_pong()`

File: commands.py [hunk 2]
```diff
@@ -469,6 +469,14 @@ def handle_who(state: server.State, user: server.UserConnection, args: List[str]
...
+    if args:
+        user.send_que.put(("PONG mantatail :" + args[0], "mantatail"))
```
Comment (ThePhilgrim) [inline commands.py:475]: I use f-strings throughout the whole code. Would you mind making this an f-string, just for consistency and my mental sanity?

**Review by ThePhilgrim — CHANGES_REQUESTED** [review]

File: commands.py (hunk 1 above)
Comment (Akuli) [reply commands.py:473]:
```suggestion
    """Handles a PING message sent from a client to the server.

    Ex:
    PING :blah blah
    :mantatail PONG mantatail :blah blah
    """
```

File: commands.py (hunk 2 above)
Comment (Akuli) [reply commands.py:480]:
```suggestion
        user.send_que.put((f"PONG mantatail :{args[0]}", "mantatail"))
```

File: commands.py [hunk 3]
```diff
@@ -469,6 +469,19 @@ def handle_who(state: server.State, user: server.UserConnection, args: List[str]
     user.send_que.put((end_of_who_message, "mantatail"))
 
 
+def handle_ping(state: server.State, user: server.UserConnection, args: List[str]) -> None:
+    """Handles a PING message sent from a client to the server.
+
+    Ex:
+    PING :blah blah
+    :mantatail PONG mantatail :blah blah
```
Comment (Akuli) [inline commands.py:477]:
```suggestion
    Ex.
        PING :This_is_a_parameter
        :mantatail PONG mantatail :This_is_a_parameter
```

File: commands.py [hunk 4]
```diff
@@ -469,6 +469,19 @@ def handle_who(state: server.State, user: server.UserConnection, args: List[str]
     user.send_que.put((end_of_who_message, "mantatail"))
 
 
+def handle_ping(state: server.State, user: server.UserConnection, args: List[str]) -> None:
+    """Handles a PING message sent from a client to the server.
```
Comment (Akuli) [inline commands.py:473]:
```suggestion
    """
    Handles a PING message sent from a client to the server.
```

**Comment by Akuli** [comment]
I think this is ready for another round of reviewing.

**Comment by ThePhilgrim** [comment]
@Akuli I haven't been on the computer for a few days. Will check this and the mantaray PR tonight!

**Review by ThePhilgrim — APPROVED** [review]

**Pull request merged**